LibraryApp/src/app/screens/BookDetails/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { shuffleArray } from '@utils/arrayUtils';
+
+import styles from './styles';
+import CommentsList from './components/CommentsList';
+import { comments } from './comments';
+import BookDetail from './components/BookDetail';
+
+class BookDetails extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { params } = navigation.state;
+    return {
+      title: params ? params.item.title : 'Unknown title for the selected book'
```

Comment:
Constants for texts! In a `constants` or `strings` file if there are more than just one or two